src/mod_maxminddb.c

```diff
@@ -312,7 +312,7 @@ static const char *set_maxminddb_env(cmd_parms *cmd,
     INFO(cmd->server, "set_maxminddb_env (server) %s %s", env, path);
 
     const int max_path_segments = 80;
-    char *path_segments[max_path_segments + 1];
+    char *path_segments[80 + 1];
```

Comment:
I suppose this could alternatively use `#define max_path_segments 80`, but that doesn't seem much better

Reply:
That's an improvement!